scripts/rhel_validation/take_kernel_version.ps1

```diff
@@ -0,0 +1,180 @@
+################################################################################
+#
+# Linux on Hyper-V and Azure Test Code, ver. 1.0.0
+# Copyright (c) Microsoft Corporation
+#
+# All rights reserved.
+# Licensed under the Apache License, Version 2.0 (the ""License"");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# THIS CODE IS PROVIDED *AS IS* BASIS, WITHOUT WARRANTIES OR CONDITIONS
+# OF ANY KIND, EITHER EXPRESS OR IMPLIED, INCLUDING WITHOUT LIMITATION
+# ANY IMPLIED WARRANTIES OR CONDITIONS OF TITLE, FITNESS FOR A PARTICULAR
+# PURPOSE, MERCHANTABLITY OR NON-INFRINGEMENT.
+#
+# See the Apache Version 2.0 License for specific language governing
+# permissions and limitations under the License.
+#
+################################################################################
+
+################################################################################
+#
+#	Description
+#
+#	This script will find out if new kernel exists for each version of Redhat
+# from RHEL6.0 to RHEL7.4.
+#	This script imports the login cookies for downloading the html to get 
+# the entire list of kernels. For each version of Redhat it creates a list with 
+# kernels associated and stores these informations in a hash table.
+#	Each version of Redhat has a latest kernel stored in a file which is compared 
+# with the last added kernel in hash table.
+#
+################################################################################
+################################################################################
+#
+# Main script body
+#
+################################################################################
+
+param (
+    [String] $WorkDir,
+    [String] $LatestVersionsFile,
+    [String] $OutputFile,
+    [String] $RemoteHtmlLocation,
+    [String] $UtilsDir
+)
+
+# generate hash table with list of kernels for each version of rhel
+$RHEL_VERSIONS_TO_KERNEL_MAP = @{"rhel_7.3" = @{"baseVer" = "3.10.0-514"; "newVer" = @()};
+                                 "rhel_7.4" = @{"baseVer" = "3.10.0-693"; "newVer" = @()}}
+
+function Get-StoredVersions {
+    param (
+        [String] $LatestVersionsFile
+    )
+    
+    if (Test-Path $LatestVersionsFile) {
+        $LatestVersions = Get-Content $LatestVersionsFile
```

Comment:
local variable, use lower caps and you can make it in one line
$latestVersions = Get-Content $LatestVersionsFile.Split(";")